lib/l10n/intl_es.arb

```
@@ -0,0 +1,198 @@
+  "createdOn": "Creado el",
+  "paymentMethodsLabel": "Métodos de pago",
+  "cancelTrade": "Cancelar Intercambio",
+  "areYouSureCancel": "¿Estás seguro de que quieres cancelar este intercambio?",
+  "cooperativeCancelMessage": "Si confirmas, iniciarás una cancelación cooperativa con tu contraparte.",
+  "acceptCancelMessage": "Si confirmas, aceptarás la cancelación cooperativa iniciada por tu contraparte.",
+  "confirm": "Confirmar",
+  "buy": "COMPRAR",
+  "sell": "VENDER",
+  "buying": "COMPRANDO",
+  "selling": "VENDIENDO",
+  "creatingNewOrder": "CREANDO NUEVA ORDEN",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)